src/App.jsx

```diff
@@ -137,13 +139,35 @@ function KanbanBoard() {
   const handleTaskUpdate = async (taskId, updates) => {
     if (!selectedTask?.columnId) return;
     await updateTask(selectedTask.columnId, taskId, updates);
+    setSelectedTask(prev => ({
+      ...prev,
+      ...updates
+    }));
   };
 
   const handleTaskDelete = async (taskId) => {
     if (!selectedTask?.columnId) return;
     await deleteTask(selectedTask.columnId, taskId);
   };
 
+  const handleAddTask = async (columnId) => {
+    if (!newTaskContent?.trim()) return;
```

Comment:
![kody code-review](https://img.shields.io/badge/kody-code--review-brightgreen)

```javascript
const handleAddTask = useCallback(async (columnId) => {
  if (!newTaskContent?.trim()) return;
```
Memoize the resetForm callback using useCallback to prevent unnecessary re-renders in child components that might receive this function as a prop.



<sub>Talk to Kody by mentioning @kody</sub>

<sub>Was this suggestion helpful? React with 👍 or 👎 to help Kody learn from this interaction.</sub><!-- kody-codereview -->&#8203;
&#8203;